spec/controllers/application_controller_spec.rb

```diff
@@ -258,6 +313,8 @@ def append_info_to_payload(payload)
             request_uuid: nil
           )
 
+          expect(Raven).not_to receive(:capture_exception)
+
```

Comment:
you can't see the lines above this, but basically whoever wrote this test probably had the expectation that capture_exception is getting called, why else would they care that extra_context is getting set otherwise? But it's worth noting that Raven is never going to report to Sentry because of the Exclusions in ExceptionHandling Mixin.

Reply:
> Raven is never going to report to Sentry

I'm confused by this, but perhaps that's my fault. I thought `Raven` was the interface to Sentry?